gammapy/utils/deprecation.py

```diff
@@ -51,3 +51,24 @@ def deprecated_attribute(name, since, **kwargs):
 
     kwargs["warning_type"] = GammapyDeprecationWarning
     return deprecated_attribute(name, since, **kwargs)
+
+
+def deprecated_key(key, old_keys, new_keys, since):
```

Comment:
In my opinion the is not needed the keys are not deprecated but the module variable `observation_locations` is.